lemur/migrations/versions/c301c59688d2_.py

```diff
@@ -31,15 +31,28 @@
 
 from alembic import op
 from sqlalchemy.sql import text
-from lemur.common import utils
 import time
 import datetime
+from flask import current_app
+
+from logging import Formatter, FileHandler, getLogger
 
-log_file = open('db_upgrade.log', 'a')
+from lemur.common import utils
+
+log = getLogger(__name__)
+handler = FileHandler(current_app.config.get("LOG_UPGRADE_FILE", "db_upgrade.log"))
```

Comment:
Thanks for making this change, much appreciated. It might be a good idea to mention about this config in the comment from top of this file ([line 10](https://github.com/Netflix/lemur/blob/master/lemur/migrations/versions/c301c59688d2_.py#L10)).
`log is added to a file named db_upgrade.log (can be overridden using LOG_UPGRADE_FILE config)`

Reply:
OK